js/ui/overlay.d.ts

```diff
@@ -83,6 +83,18 @@ export interface dxOverlayOptions<TComponent> extends WidgetOptions<TComponent>
      * @public
      */
     hideOnParentScroll?: boolean;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    preventScrollEvents?: boolean;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    enableBodyScroll?: boolean;
```

Comment:
Are u sure we should publish it on overlay level?
I think it's better to keep it private in overlays, but publish it in Popup with a default = false

Reply:
I made it public for Popup. Popover enherit this doc tag and will public too. In Tooltip I made it hidden.